lib/portage/eapi.py

```diff
@@ -3,6 +3,7 @@
 
 import collections
 import logging
```

Comment:
Should we maybe do `from logging import logging.warning` or something instead?

Reply:
I tend to prefer just the module, since it provides a form of documentation in the code itself regarding where functions/constants come from.

Are you thinking it's more lightweight to import just the one constant?

I dug into it a bit, and [this SO](https://softwareengineering.stackexchange.com/a/187471) seems to suggest that the same import occurs, just with different semantics.

Reply:
> Are you thinking it's more lightweight to import just the one constant?

Objectively speaking, using `logging.WARNING` in a few places is not going to have any real performance hit. It only matters when the dot operator does multiple lookups during a loop, especially very large loops. For single use statements, what @ezzieyguywuf did is fine and is more explicit too.

Reply:
I'd got confused -- the original issue wasn't an import-time thing, it was to do with logging and implicit logger name. So yeah, all fine!